wandb/sdk/artifacts/artifact.py

```diff
@@ -532,6 +532,27 @@ def type(self) -> str:
         """The artifact's type. Common types include `dataset` or `model`."""
         return self._type
 
+    @property
+    def url(self):
+        """
+        Constructs the URL of the artifact.
+
+        Returns:
+            str: The URL of the artifact.
+        """
+        if self.collection.is_sequence():
+            return f"{self._client.app_url}{self._entity}/{self._project}/artifacts/{quote(self._type)}/{quote(self.collection.name)}/{self._version}"
+        else:
+            if self._project.startswith("wandb-registry-"):
+                org = InternalApi()._fetch_orgs_and_org_entities_from_entity(
+                    self._entity
+                )[0]
+                return f"{self._client.app_url}orgs/{org.display_name}/registry/{self._type}?selectionPath={self._entity}%2F{self._project}%2F{self.collection.name}&view=membership&version={self._version}"
+            elif self._type == "model" or self._project == "model-registry":
+                return f"{self._client.app_url}{self._entity}/registry/model?selectionPath={self._entity}%2F{self._project}%2F{self.collection.name}&view=membership&version={self._version}"
```

Comment:
lets use `quote(<val>, safe='')` here instead of directly adding `'%2F'`. `quote('/', safe='')` quotes `'/'` as `'%2F'` 

the backend generates these urls by "quoting" after the `?`, i.e., the query part of the url.

Reply:
Done!